.github/workflows/build.yml

```diff
@@ -203,8 +234,8 @@ jobs:
         run: uv run -- mkdocs build --strict
 
   # An optional job to alert us when uv updates break the build
-  mkdocs-latest:
-    name: Check that MkDocs can build the docs with latest Python and uv
+  check-mkdocs-latest:
+    name: check (mkdocs latest)
```

Comment:
I feel like this rename lost a lot of info. I'm not sure what's best, but let's keep `Check that MkDocs can build the docs` and `Check that MkDocs can build the docs with latest Python and uv` as a comment, at least.

Perhaps there's a better way to share this info, e.g. put it into some sort of "description" field or print it out, but I think a comment is sufficient as a minimal solution.

Reply:
What even is this testing? What's the backstory on this entry? Could we just always build with the latest version and drop *the other* version?

Reply:
There are two reasons for this, the first one being the main one:

- if `uv` (or Python, I guess) has a breaking change, I want to know about it, but I do not want PR authors to be blocked by it. So, I want the "required" check to use a fixed version of everything, and have a non-required check (this one) that checks the latest versions of everything.
- We say we support `uv 0.5.1+` in contributing.md, and I want there to be a check verifying this. As time goes by, I imagine more and more people will be on a non-latest version of `uv`.

Reply:
Perhaps this will be a bit clearer if you add "non-required" to the name of the check. This will also make it easier for Martin (or for future updates of the rules by anyone) to make the correct check required and leave this one optional.

Reply:
Fixed. TBH I feel like having two checks is honestly a bit redundant in a way, but maybe we can address that later; I'd like to get this in sooner than that, though :)